### PR TITLE
Issue 10: Research overview discovery

### LINKED ISSUE #10: New search result view: Research Overview
Result: full text search of "research overview" only

### PR DESCRIPTION
**Pull request by wwelling**
Resolves #10

### CODE REVIEW

File: solr/config/solrconfig.xml [hunk 1]
```diff
@@ -1180,7 +1180,6 @@
       <str>yyyy-MM-dd HH:mmZZ</str>
 
       <str>yyyy-MM-dd</str>
-      <str>yyyy</str>
```
Comment (kaladay) [inline solr/config/solrconfig.xml:1183]: This was just added as of #25 .
Why remove it?

File: src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java [hunk 2]
```diff
@@ -13,46 +13,64 @@
...
     @Value("${spring.data.solr.operator:AND}")
-    private Operator queryOperator;
+    private Operator defaultOperator;
 
     @Autowired
     private SolrTemplate solrTemplate;
 
+    @PostConstruct
+    public void setup() {
+        // https://jira.spring.io/browse/DATASOLR-153
```
Comment (kaladay) [inline src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java:68]: I think you should update the README in regards to these.
At least until they accept and merge the changes.

File: solr/config/solrconfig.xml (hunk 1 above)
Comment (wwelling) [reply solr/config/solrconfig.xml:1183]: Shouldn't of been added. The date object requires month and day. It was effectively doing nothing until trying to index a pdate with year only.

File: src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java (hunk 2 above)
Comment (wwelling) [reply src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java:68]: Not a bad idea.

Comment (wwelling) [reply src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java:68]: Done.

**Pull request merged**